js/interactive-guides/step-content.js

```diff
@@ -262,6 +262,35 @@ var stepContent = (function() {
     createEndOfGuideContent();
   };
 
+  var calculateWidgetHeight = function(percentageHeight) { 
+    var widgetHeight = "300px";
+    var rightColumn = $("#code_column:visible");
+    if (rightColumn) {
+        // Get height of visible right column
+        var columnHeight = $(rightColumn).height();
+        var wHeight = (columnHeight * percentageHeight)/100;
+        widgetHeight = wHeight + "px";
+    }
+    return widgetHeight;
+  }
+
+  // we could have a 2 widgets or 3 widgets on the right column
+  // base on the number of widgets determined the percentage height
+  // if 3 widgets: browser: 40% height, pod: 20% height, editor: 40% height
+  var getWidgetPercentageHeight = function(numOfWidgets, widgetType) {
+    var defaultPercentageHeight = 50;
+    if (numOfWidgets === 2) {
+        return defaultPercentageHeight;
+    } else if (numOfWidgets === 3) {
+        if (widgetType === "tabbedEditor" || widgetType === "webBrowser") {
+            return 40;
+        } else if (widgetType === "pod") {
```

Comment:
I thought we were going to leave the pod height a constant size so we would never add scroll bars to it?  Since the pod for the other guides is so small it may not need 20% of the height.  Or is this all based on the new design?